scripts/migration_check

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ -d './db/migrate' ]
+then
+	echo "The ./db/migrate folder should not be used for generating migrations"
+	echo "Use the --database option to specify the database to be used. Example:"
+	echo "rails g migration add_my_database_table --database primary"
+	exit 1
```

Comment:
since this is is an "exists" and not an "is empty" check, maybe add instructions to remove the directory too?

```suggestion
	echo "The ./db/migrate folder should not be used for generating migrations"
	echo "Use the --database option to specify the database to be used. Example:"
	echo "> rails g migration add_my_database_table --database primary"
	echo "Then, remove db/migrate"
	exit 1
```

Reply:
Yep makes sense, thanks